.gitignore

```diff
@@ -0,0 +1,23 @@
+build/*
+.idea/*
```

Comment:
_nit_: IMO this (and other IDE-specific things) belongs in a user's global `.gitignore` file and not checked into the repo.